lib/features/search/search.js

```diff
@@ -175,10 +175,16 @@ function scoreToken(token) {
   }
 
   return token.start
-    ? 1.37
-    : token.wordStart
-      ? 1.13
-      : 1;
+    ? (
+      token.end
+        ? 131.9
+        : 7.87
+    )
+    : (
+      token.wordStart
+        ? 2.19
+        : 1
```

Comment:
What magic is this? :) Why such values?

Reply:
Standard magic values, call them temperature :hot_face:

Reply:
Gives me AI engineer vibes :D

Reply:
You gotta tune it some times. Though we have test coverage, which is good. :wink: